fdbserver/include/fdbserver/DataDistribution.actor.h

```diff
@@ -476,6 +476,20 @@ struct ShardSizeBounds {
 	bool operator==(ShardSizeBounds const& rhs) const {
 		return max == rhs.max && min == rhs.min && permittedError == rhs.permittedError;
 	}
+
+	static ShardSizeBounds shardSizeBoundsBeforeTrack() {
```

Comment:
Nit: The implementation of this function should be moved out of the header file.